docs/x509.rst

```diff
@@ -346,6 +366,75 @@ instances. The following common OIDs are available as constants.
     Corresponds to the dotted string ``"1.2.840.113549.1.9.1"``. This OID is
     typically seen in X.509 names.
 
+Signature Algorithm OIDs
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. data:: OID_MD5_WITH_RSA
+
+    Corresponds to the dotted string ``"1.2.840.113549.1.1.4"``. This is
+    an MD5 digest signed by an RSA key.
+
+.. data:: OID_SHA1_WITH_RSA
+
+    Corresponds to the dotted string ``"1.2.840.113549.1.1.5"``. This is
+    a SHA1 digest signed by an RSA key.
+
+.. data:: OID_SHA224_WITH_RSA
+
+    Corresponds to the dotted string ``"1.2.840.113549.1.1.14"``. This is
+    a SHA224 digest signed by an RSA key.
+
+.. data:: OID_SHA256_WITH_RSA
+
+    Corresponds to the dotted string ``"1.2.840.113549.1.1.11"``. This is
+    a SHA256 digest signed by an RSA key.
+
+.. data:: OID_SHA384_WITH_RSA
+
+    Corresponds to the dotted string ``"1.2.840.113549.1.1.12"``. This is
+    a SHA384 digest signed by an RSA key.
+
+.. data:: OID_SHA512_WITH_RSA
+
+    Corresponds to the dotted string ``"1.2.840.113549.1.1.13"``. This is
+    a SHA512 digest signed by an RSA key.
+
+.. data:: OID_ECDSA_WITH_SHA224
```

Comment:
So, I think we should make the names at this level in Python consistent
with each other, and not with teh spec.

On Wed, Feb 18, 2015 at 8:39 PM, Paul Kehrer <notifications@github.com>
wrote:

> In docs/x509.rst
> <https://github.com/pyca/cryptography/pull/1651#discussion_r24959970>:
>
> > +.. data:: OID_SHA256_WITH_RSA
> > +
> > +    Corresponds to the dotted string ``"1.2.840.113549.1.1.11"``. This is
> > +    a SHA256 digest signed by an RSA key.
> > +
> > +.. data:: OID_SHA384_WITH_RSA
> > +
> > +    Corresponds to the dotted string ``"1.2.840.113549.1.1.12"``. This is
> > +    a SHA384 digest signed by an RSA key.
> > +
> > +.. data:: OID_SHA512_WITH_RSA
> > +
> > +    Corresponds to the dotted string ``"1.2.840.113549.1.1.13"``. This is
> > +    a SHA512 digest signed by an RSA key.
> > +
> > +.. data:: OID_ECDSA_WITH_SHA224
>
> Isn't that infuriating? Sadly that's their name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1651/files#r24959970>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084